Kernel/FileSystem/InodeFile.cpp

```diff
@@ -96,6 +96,8 @@ ErrorOr<NonnullOwnPtr<KString>> InodeFile::pseudo_path(OpenFileDescription const
 ErrorOr<void> InodeFile::truncate(u64 size)
 {
     TRY(m_inode->truncate(size));
+    // FIXME: In most cases, we already updated the timestamps in Inode::truncate.
+    //        However, we should probably do it here as well, to ensure that the timestamps are always updated.
```

Comment:
What's the FIXME exactly? We already update the timestamps two lines down.

Reply:
So, should I change the `FIXME` into a `NOTE`? Or write something like "Make sure that the timestamps are updated by Inode::truncate for all filesystems before removing it here"?

Reply:
That last one sounds fine!